kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/TopicsResourceIntegrationTest.java

```diff
@@ -184,4 +185,71 @@ public void getTopic_nonExistingTopic_returnsNotFound() {
         request("/v3/clusters/" + clusterId + "/topics/foobar").accept(Versions.JSON_API).get();
     assertEquals(Status.NOT_FOUND.getStatusCode(), response.getStatus());
   }
+
+  @Test
+  public void createTopic_nonExistingTopic_returnsCreatedTopic() throws Exception {
+    String baseUrl = restConnect;
+    String clusterId = getClusterId();
+    String topicName = "topic-4";
```

Comment:
if you manually manage a kafka cluster to run these tests, using a uuid for topic name would be more convenient.

Reply:
These integration tests are run against an in-process Kafka instance, which is created and destroyed by the tests. The test is hermetic, that's why I don't have to introduce randomness.